opencraft/settings.py

```diff
@@ -114,7 +114,8 @@
     'registration',
     'reports',
     'backup_swift',
-    'periodic_builds'
+    'periodic_builds',
+    'e2e_tests',
```

Comment:
Do we need to have this configured for all runtimes including production?
It seems like something we wouldn't want to add to the installed apps on runtime systems like production and stage, just while running tests.

Maybe we can split the test settings out into a separate settings file where this is added?

Reply:
This makes sense @kaizoku  and I think splitting the settings file might go outside the scope of this PR.

I think the following approach works with minimal changes.

```python
# This env var is already added for e2e tests, we can utilise this
E2E_TESTS = env.bool('E2E_TESTS', default=True)
...
# If we are running e2e test add e2e_tests application in LOCAL_APPS
if E2E_TESTS:
    LOCAL_APPS += (
        'e2e_tests',
    )
```

Reply:
That works as well @arjunsinghy96. Thanks for the update to that.